otter/integration/tests/test_convergence.py

```diff
@@ -183,6 +273,25 @@ def _choose_half_the_servers(self, (code, response)):
         self.n_killed = self.n_servers / 2
         return ids[:self.n_killed]
 
+    def _choose_servers_from_active_list(self, (code, response),
+                                         num_servers=None):
+        """Select the first 'num_servers' of the servers returned by the
+        ``get_scaling_group_state`` function.  Record the number of servers
+        received in ``active_servers`` attribute. If no num_servers is
+        provided, all active servers will be returned.
+
+        If the number of servers requested is less than the number active,
+        an error will be raised.
+        """
+
+        if code != 200:
```

Comment:
I was thinking that 404 is not found, but what if we got back a 403, 500, or other? Is there a reason for checking the 404 explicitly and letting the rest propagate?